### PR TITLE
Documentation for Partial and Downstream Pipelines in CI Visibility

### PR DESCRIPTION
**Pull request by kassenq**
### What does this PR do?
This PR adds documentation for partial and downstream pipelines in CI Visibility, which are now supported for the CI providers Buildkite and GitLab.

### Motivation
Let customers know which tags they can use for grouping and filtering.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/pipelines/buildkite.md [hunk 1]
```diff
@@ -83,6 +83,17 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/buildkite.md:87]:
```suggestion
In the **Pipeline Executions** page, you can use the filters below in the search bar:
```

File: content/en/continuous_integration/pipelines/buildkite.md [hunk 2]
```diff
@@ -83,6 +83,17 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
...
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
+
+{{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}
+
+These filters can also be applied via the facet panel on the left hand side of the page.
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/buildkite.md:94]:
```suggestion
These filters can also be applied through the facet panel on the left hand side of the page.
```

File: content/en/continuous_integration/pipelines/buildkite.md [hunk 3]
```diff
@@ -83,6 +83,17 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
+
+{{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/buildkite.md:92]:
```suggestion
{{< img src="ci/partial_retries_search_tags.png" alt="The Pipeline executions page with Partial Pipeline:retry entered in the search query" style="width:100%;">}}
```

Comment (buraizu) [reply content/en/continuous_integration/pipelines/buildkite.md:92]: Just following up on this image:
- Is the Pipeline Executions page the correct place to enter this search? The UI of that page doesn't seem to match your screenshot. Perhaps this is a limitation of the demo account?
- I updated the alt text with our internal guidelines in mind

File: content/en/continuous_integration/pipelines/buildkite.md [hunk 4]
```diff
@@ -83,6 +83,17 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
...
+
+These filters can also be applied via the facet panel on the left hand side of the page.
+{{< img src="ci/partial_retries_facet_panel.png" alt="GitLab Partial Retry Facet Panel" style="width:40%;">}}
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/buildkite.md:95]:
```suggestion
{{< img src="ci/partial_retries_facet_panel.png" alt="The facet panel with Partial Pipeline facet expanded and the value Retry selected, the Partial Retry facet expanded and the value true selected" style="width:40%;">}}
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 5]
```diff
@@ -141,6 +141,18 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:145]:
```suggestion
In the **Pipeline Executions** page, you can use the filters below in the search bar:
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 6]
```diff
@@ -141,6 +141,18 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
...
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
+
+{{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:150]:
```suggestion
{{< img src="ci/partial_retries_search_tags.png" alt="The Pipeline executions page with Partial Pipeline:retry entered in the search query" style="width:100%;">}}
```

File: content/en/continuous_integration/pipelines/gitlab.md (hunk 5 above)
Comment (buraizu) [reply content/en/continuous_integration/pipelines/gitlab.md:145]: Same question as the other doc regarding this screenshot

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 7]
```diff
@@ -141,6 +141,18 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:148]:
```suggestion
`Downstream Pipeline`
: Possible values: `true`, `false`

`Manually Triggered`
: Possible values: `true`, `false`

`Partial Pipeline`
: Possible values: `retry`, `paused`, `resumed`
```

Comment (buraizu) [reply content/en/continuous_integration/pipelines/gitlab.md:148]: Suggesting to incorporate this content as a description list for added clarity.

File: content/en/continuous_integration/pipelines/buildkite.md [hunk 8]
```diff
@@ -83,6 +83,17 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/buildkite.md:90]:
```suggestion
`Downstream Pipeline`
: Possible values: `true`, `false`

`Manually Triggered`
: Possible values: `true`, `false`

`Partial Pipeline`
: Possible values: `retry`, `paused`, `resumed`
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 9]
```diff
@@ -141,6 +141,18 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
...
+
+{{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}
+
+These filters can also be applied via the facet panel on the left hand side of the page.
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:152]:
```suggestion
These filters can also be applied through the facet panel on the left hand side of the page.
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 10]
```diff
@@ -141,6 +141,18 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
...
+{{< img src="ci/partial_retries_facet_panel.png" alt="GitLab Partial Retry Facet Panel" style="width:100%;">}}
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:153]:
```suggestion
{{< img src="ci/partial_retries_facet_panel.png" alt="The facet panel with Partial Pipeline facet expanded and the value Retry selected, the Partial Retry facet expanded and the value true selected" style="width:40%;">}}
```

Comment (buraizu) [reply content/en/continuous_integration/pipelines/gitlab.md:153]: It looks like this screenshot would also benefit from a lower width setting

**Review by buraizu — CHANGES_REQUESTED** [review]

File: content/en/continuous_integration/pipelines/buildkite.md (hunk 3 above)
Comment (kassenq) [reply content/en/continuous_integration/pipelines/buildkite.md:92]: Yes, the Pipeline Executions page is the correct place for the search, but the demo account does not contain a lot of our newest features. Updating our demo environment is something we are actively working on :)

File: content/en/continuous_integration/pipelines/gitlab.md (hunk 10 above)
Comment (kassenq) [reply content/en/continuous_integration/pipelines/gitlab.md:153]: As long as the text in the screenshot is not too small to see!

File: content/en/continuous_integration/pipelines/buildkite.md [hunk 11]
```diff
@@ -83,6 +83,22 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the **Pipeline Executions** page, you can use the filters below in the search bar:
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/buildkite.md:87]:
```suggestion
### Partial and downstream pipelines

In the **Pipeline Executions** page, you can use the filters below in the search bar:

```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 12]
```diff
@@ -141,6 +141,23 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:145]:
```suggestion
### Partial and downstream pipelines

In the **Pipeline Executions** page, you can now use the filters below in the search bar:

```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for implementing those changes, and apologies -- I just noticed in the preview that with the current formatting, the first line of the content is accidentally included in the description list. The suggestion made here resolve that, and also make the **Pipelines Executions** callout consistent between the two pages.

File: content/en/continuous_integration/pipelines/gitlab.md (hunk 12 above)
Comment (kassenq) [reply content/en/continuous_integration/pipelines/gitlab.md:145]: No worries, thank you!

**Pull request merged**